app/models/concerns/sufia/file_set_behavior.rb

```diff
@@ -2,5 +2,10 @@ module Sufia
   module FileSetBehavior
     extend ActiveSupport::Concern
     include Sufia::WithEvents
+
+    # Cast to a SolrDocument by querying from Solr
+    def to_presenter
+      CatalogController.new.fetch(id).last
```

Comment:
This is clever. Never seen it before... Is this a Blacklight thing or a Rails thing?